mpt/impl/imap-mailbox/jpa/src/test/java/org/apache/james/mpt/imapmailbox/jpa/JpaSelectedStateTest.java

```diff
@@ -56,4 +56,9 @@ public void testUidKOREA() {
     @Override
     public void testUidUS() {
     }
+
+    @Override
+    public void testSearchSaveDate() {
+        // SEARCH save date just return naive/empty result for JPA.
+    }
```

Comment:
`@Ignore("SEARCH save date just return naive/empty result for JPA")`

Also, generally search indexes might need a delay between append and search to have the time to index data.